libraries/AP_DDS/README.md

```diff
@@ -167,7 +167,6 @@ After your setups are complete, do the following:
 Unlike the use of ROS 2 `.msg` files, since Ardupilot supports native DDS, the message files follow [OMG IDL DDS v4.2](https://www.omg.org/spec/IDL/4.2/PDF).
 This package is intended to work with any `.idl` file complying with those extensions, with some limitations. 
 
-1. IDL files need to be in the same folder, and modified includes.
 1. Topic types can't use alias types.
```

Comment:
Can you also remove these two limitations, and the caveat above there being limitations, since there are no longer limitations.

Reply:
no problem !

Reply:
Done and thanks for your help !